cpp/src/ice2slice/Gen.cpp

```diff
@@ -275,28 +275,27 @@ namespace
         }
     }
 
-    string slice2LinkFormatter(const string& identifier, const string& memberComponent)
+    string slice2LinkFormatter(string rawLink, const ContainedPtr&, const SyntaxTreeBasePtr&)
     {
-        // Replace links of the form `{@link Type#member}` with `{@link Type::member}`.
-        string result = "{@link ";
-        if (memberComponent.empty())
+        // The only difference with '@link' between Slice1 and Slice2 is that Slice1 uses '#' and Slice2 uses '::'.
```

Comment:
I assume you mean the `.ice` and `.slice` syntax, not Slice1 and Slice2. Slice1 and Slice2 are two modes of the .slice syntax.

Reply:
We can also say `Ice Slice files (.ice) and IceRPC Slice files (.slice)` but is kind of verbose.

Reply:
This is a comment in the ice2slice implementation so you could write:
> The only difference with '@link' between the ice and slice syntaxes is that the ice syntax uses '#' ...

Reply:
I went with 'Ice syntax' and 'Slice syntax' because I don't care enough to argue.
Of course, I'm still going to make my case though : vP

Because these terms feel pretty weird to me.
There is no such thing as 'Ice syntax' and 'Slice syntax' is ambiguous IMO.
Whereas 'Slice1' and 'Slice2' make it clear that we're only talking about the Slice syntax, but just two different versions of it.

It happens to be true that 'Slice1' and 'Slice2' are compilation modes in IceRPC, but that has no bearing here.
Ice has no compilation modes. And it's worth noting that the entire point of 'Slice1's existence is to interop with the 'Ice syntax'. So calling this 'Slice1' feels very natural to me. </case>